### PR TITLE
Teaching style nov 2020 hpc

### PR DESCRIPTION
**Pull request by Sabryr**
Customization for the HPC trainer training event

### CODE REVIEW

**Comment by rkdarst** [comment]
This looks good, but some suggestions:

I wouldn't customize it too much for HPC carpentry but try to make it generic so we don't have to do much extra work later.  We could leave HPC-carpentry specific stuff in an admonition like this:
``````
```{admonition} HPC carpentry
hpc-specific note here
````
``````
so that it's easy to find and edit/remove/comment out them later.  Same with not removing stuff that would be otherwise useful for a broader CodeRefinery audience - we can say some of these things don't apply.  But when there are lists of things, I'd say it's OK to leave them in, since lists can be long and labeled like they are now.

**Comment by Sabryr** [comment]
@rkdarst I wanted to check how "admonition" renders in my folk, but I could not get the proper settings to work
https://sabryr.github.io/instructor-training/
Do you have notes on setting up the sphinx on Github?

**Comment by rkdarst** [comment]
Can you point to the specific problem?  That rendering looks OK on your gh-pages, and it looks like you got sphinx working properly.  Admonition with at title should be like this:
`````
```{admonition} Title
content
````
`````

Ones with a pre-designated title are like this (a first line isn't a title):
`````
```{note}
content
````
`````

I think you could merge this and then we follow up with anything remaining later.

**Comment by Sabryr** [comment]
Thanks, it was a misundestanding, I was expecting a collapseble content in rendering. Yes I would have to make a request to merge this and come back to this after the workshop.

**Comment by rkdarst** [comment]
Ah, I see.  A `solution` block is automatically collapsable (sphinx-lesson setting):

`````
```{solution} (optional title)
content
````
`````

Generic is using [sphinx-togglebutton](https://github.com/executablebooks/sphinx-togglebutton) (this example is untested):

`````
```{admonition}
:class: dropdown

content
```
`````

File: content/03-teaching-style.md [hunk 1]
```diff
@@ -130,96 +158,127 @@ The first task in teaching is to figure out who your learners are. The Carpentri
...
 
-> ## Exercise: Carpentries versus CodeRefinery audience
+> ## Exercise: How to identify learner profiles?
 >
-> Split into groups (3 to 4 persons). Discuss and summarize in the hackmd pad:
-> 1. Why does CodeRefinery target **competent practitioner**?
-> 2. Why is it important to be able to write code for attending a CodeRefinery workshop?
-> For this exercise, we suggest to read [Motivation and Demotivation](https://carpentries.github.io/instructor-training/08-motivation/index.html) from the Carpentries instructor training lesson.
+> 1. How to identify leaner profiles from surveys and during the class
+> 2. Which types of leaners should the leassons focus on
```
Comment (rkdarst) [inline content/03-teaching-style.md:258]:
```suggestion
> 2. Which types of learners should the leassons focus on
```

**Review by rkdarst — APPROVED** [review]
Looks good, one small thing, but I will go on and see what else to do next.

**Pull request merged**